programs/lfx-mentorship/2024/02-Jun-Aug/project_ideas.md

```diff
@@ -188,3 +188,22 @@ Onboarding new end users into a sophisticated system like Knative Eventing prese
     - Frances Thai @notfelineit frances@planetscale.com
 - Upstream Issue (URL): https://github.com/vitessio/arewefastyet/issues/525
 
+### KWOK
+
+#### Enhancement of Test Cases
+
+- Description: KWOK is currently being used by a number of projects for testing and performance. It is crucial that KWOK itself behaves consistently. The following tests are currently being considered: Unit Test, E2E Test, Edge Cases.
```

Comment:
```suggestion
- Description: KWOK is a foo bar foo bar foo bar. KWOK  is currently being used by a number of projects for testing and performance. It is crucial that KWOK itself behaves consistently. The following tests are currently being considered: Unit Test, E2E Test, Edge Cases.
```
Can you add a few words about what KWOK is please? That would increase your chances of finding a mentee.